pynta/utils.py

```diff
@@ -113,6 +114,11 @@ def get_unique_sym_struct_indices(geoms):
 
     for i,geom in enumerate(geoms_copy):
         adsorbate_atom_obj = geom
+        for constraint in adsorbate_atom_obj.constraints:
```

Comment:
We do this here for only one of the structure comparison functions, but shouldn't we have this in all of them?

Reply:
Yeah probably. I can adjust them all after I figure out whether or not we want to add slab constraints before comparison.